spec/plans/execute.fmf

```diff
@@ -318,3 +291,22 @@ description: |
                     systemctl start httpd
                     echo foo > /var/www/html/index.html
                     curl http://localhost/ | grep foo
+
+/sync:
+    summary: Synchronization between steps or tests
+    description:
+        The main idea of :ref:`/spec/plans/provision/multihost`
+        tests is to run certain tasks in parallel (e.g., a RestAPI
+        requests from a client to a server). Therefore, these kinds
+        of tests may require test synchronization.
+        The synchronization may be specified for specific steps,
+        tests or for all of them using the `each` keyword.
+
+    example: |
+        execute:
+            how: tmt
+            sync:
+                step:
+                  - prepare
+                  - execute
+                test: each
```

Comment:
I think it would be good to add a few more words describing the expected options for both `step` and `test` keys. For example for the `test` sync we considered/brainstormed on the hacking session `test-name`, `index` and `none`.

Reply:
The `sync` definition is extended, please have a look.